### PR TITLE
Adding a cmdline_upgrade

### PR DESCRIPTION
**Pull request by m2bright**

### CODE REVIEW

File: rapid/__main__.py [hunk 1]
```diff
@@ -37,7 +38,7 @@
 parser.add_argument('--generate-config', dest='generate_config', help='Generate a default configuration', choices=['master', 'client'])
 args = parser.parse_args()
 
-if args.mode_client or args.run:
+if args.mode_client or args.run or args.upgrade:
```
Comment (BambooBryson) [inline rapid/__main__.py]: you don't care if it's a run or an upgrade?

File: rapid/client/client_upgrader.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        communicator = ClientCommunicator(self.client_config.master_uri, self.client_config.quarantine_directory, app, self.client_config.verify_certs)
+        self.client_config.is_single_use = True
+
+        info = communicator.register(self.client_config)
+        if info['master_version'] != Version.get_version():
+            logger.info("Found Version: {}".format(info['master_version']))
+            # UpgradeUtil.upgrade_version(info['master_version'], self.client_config)
+            UpgradeUtil.upgrade_version(info['master_version'], self.client_config)
+        else:
+            logger.info("Versions match. No need to upgrade")
```
Comment (BambooBryson) [inline rapid/client/client_upgrader.py]: Should this be wrapped in a try? In the event that the key might not exist?

Comment (m2bright) [reply rapid/client/client_upgrader.py]: Great question! Since this is a command line execution, I want it to error, if there is no key.

Comment (BambooBryson) [reply rapid/client/client_upgrader.py]: great point... lol Thanks!

**Pull request merged**